packages/terra-switch/README.md

```diff
@@ -0,0 +1,45 @@
+# Terra Switch
+
+[![NPM version](https://badgen.net/npm/v/terra-switch)](https://www.npmjs.org/package/terra-switch)
+[![Build Status](https://badgen.net/travis/cerner/terra-switch)](https://travis-ci.com/cerner/terra-core)
+
+The terra-switch component indicates whether a setting is on or off and provide users with instantaneous feedback.
+
+Switches are used to activate settings immediately, in real time.
+Switches never require users to press a button to apply their settings.
+A switch is a set of binary options and always has one of the options selected by default.
+
+Switch should be used for only one of these set of binary options:
+• On / Off
```

Comment:
```suggestion
The terra-switch component indicates whether a setting is ON or OFF and provide users with instantaneous feedback.
```
This description can be shortened, since we need not mention the use of the component and also majority of the information can be found in the props table

Reply:
formated readme and Doc : 48fce2d9ca73773ac6f9891feaa7f2a56b36f864